docs/rfc-1-staking-contract.adoc

```diff
@@ -421,20 +431,22 @@ stake for that provider or if amount is higher than the cached legacy stake
 amount. If succeeded, the legacy NU stake can be partially or fully undelegated
 on the legacy staking contract. This function allows to unstake from NU staking 
 contract and sill being able to operate in T network and earning rewards based
-on the liquid T staked. Can be called only by the delegation owner and provider.
+on the liquid T staked. Can be called only by the delegation owner or the
+staking provider.
 
 ==== `unstakeAll(address stakingProvider) external onlyOwnerOrStakingProvider(stakingProvider)`
 
 Sets cached legacy stake amount to 0, sets the liquid T stake amount to 0 and
 withdraws all liquid T from the stake to the owner. Reverts if there is at least one
-non-zero authorization. Can be called only by the delegation owner and provider.
+non-zero authorization. Can be called only by the delegation owner or the
+staking  provider.
```

Comment:
```suggestion
non-zero authorization. Can be called only by the delegation owner or the
staking provider.
```

Reply:
Addressed in #72 @ https://github.com/threshold-network/solidity-contracts/pull/72/commits/872717194a6c70898da7eb1b14db8b6700f35e5f